packages/block-library/src/image/interactivity.js

```diff
@@ -22,6 +22,11 @@ store( {
 		core: {
 			image: {
 				showLightbox: ( { context, event } ) => {
+					// We can't initialize the lightbox until the reference
+					// image is loaded, otherwise the UX is broken.
+					if ( ! context.core.image.imageLoaded ) {
+						return;
+					}
```

Comment:
I hope time to interactive does not being affected here too much.

Reply:
It depends on the user's bandwidth. But also, I think it's reasonable to prevent a user from zooming in on an image that hasn't fully loaded yet? 🤔

Another note: We can likely make the time to interactivity instant with the fade animation; it's just the zoom animation that causes the trouble.

If it's worth exploring, I could make this initialization step specific to the zoom animation.

Another note: Is the concern here that users may want to go immediately to the zoomed image without needing to wait for the rest of the page to load? @mtias has [mentioned before](https://core.trac.wordpress.org/ticket/57913#comment:27) the idea of having a separate URL for the zoomed version of the image inside of the lightbox as an overall usability improvement for WordPress sites.

Perhaps we can continue to explore this in a subsequent PR where we start to identify how this would behave, (what should the URL be, how to implement the experience of initializing the page with the lightbox already open, crossover or not with the Media page, etc.)

Reply:
> It depends on the user's bandwidth. But also, I think it's reasonable to prevent a user from zooming in on an image that hasn't fully loaded yet? 🤔

It is indeed. But, for example, IGDB on their Lightbox, shows a loading text if the image is not fully loaded. Still, is something that we can explore in future PRs. 

> Is the concern here that users may want to go immediately to the zoomed image without needing to wait for the rest of the page to load?

Yes, this could happen. Usually, clicking on an image and wait for the zoom to appear is not ideal.

We can wait and see how that ticket is evolving.